distribution/test/itests/test-itests-catalog/pom.xml

```diff
@@ -70,6 +70,18 @@
             <scope>test</scope>
         </dependency>
         <dependency>
+            <groupId>org.apache.httpcomponents</groupId>
+            <artifactId>com.springsource.org.apache.httpcomponents.httpcore</artifactId>
+            <version>4.2.1</version>
```

Comment:
ddf-platform's pom uses version 4.3.3 of `httpcore` and 4.3.5 of `httpclient`. Is there a reason you're specifying older versions here?